code/modules/unit_tests/craftable_clothes.dm

```diff
@@ -28,6 +28,7 @@ abstract types are automatically excluded.
 		/obj/item/clothing/head/helmet/skullcap/cult, // cultist item
 		/obj/item/clothing/head/helmet/leather/saiga, // idk what kind of recipe to make this
 		/obj/item/clothing/neck/mana_star, // todo?
+		/obj/item/clothing/wrists/gem, // this is not supposed to be craftable as it is a base for other items
```

Comment:
mark it as abstract instead, abstract_type = {its typepath} same for say crafting/.../jade or carvedgem/jade